techniques/system/common/1.0/failsafe.st

```diff
@@ -64,20 +64,13 @@ bundle common g
       # We would like to use date's "--rfc-3339=second" option here, but it is not available on older OSes (RHEL 3/4, AIX 5...)
       "execRun"                  string => execresult("/bin/date -u \"+%Y-%m-%d %T+00:00\"", "noshell");
 
-# definition of the node roles
-  &NODEROLE&
```

Comment:
this will be a problem on relay: it's the NODEROLE that says that it's a policy server

Reply:
this information does not seem to be actually used in failsafe